newrelic_lambda_cli/integrations.py

```diff
@@ -127,30 +127,28 @@ def create_parameters(
     capabilities = ["CAPABILITY_IAM"]
     if role_name is not None:
         parameters.append({"ParameterKey": "FunctionRole", "ParameterValue": role_name})
-        capabilities = []
     elif mode != "CREATE":
         parameters.append({"ParameterKey": "FunctionRole", "UsePreviousValue": True})
-        capabilities = []
 
     return parameters, capabilities
 
 
 def import_log_ingestion_function(
-    session, nr_license_key, enable_logs, memory_size, timeout, role_name,
+    session, nr_license_key, enable_logs, memory_size, timeout, role_name
 ):
     parameters, capabilities = create_parameters(
         nr_license_key, enable_logs, memory_size, timeout, role_name, "IMPORT"
     )
     cf_client = session.client("cloudformation")
 
-    click.echo(f"Fetching new CloudFormation template url")
+    click.echo("Fetching new CloudFormation template url")
 
     template_path = os.path.join(
-        os.path.dirname(os.path.abspath(__file__)), "templates", "import-template.yaml",
+        os.path.dirname(os.path.abspath(__file__)), "templates", "import-template.yaml"
     )
     with open(template_path) as template:
-        change_set_name = f"{INGEST_STACK_NAME}-IMPORT"
-        click.echo(f"Creating change set {change_set_name}")
+        change_set_name = "%s-IMPORT" % INGEST_STACK_NAME
+        click.echo("Creating change set: %s" % change_set_name)
```

Comment:
Why the change to old-style string formatting? Is there something I don't know about Python interpolation?

Reply:
Yes, f-strings are a 3.6+ feature. The CLI currently supports 3.3+.